pvlib/tests/iotools/test_bsrn.py

```diff
@@ -85,13 +85,17 @@ def test_get_bsrn(expected_index, bsrn_credentials):
         station='tam',
         username=username,
         password=password,
-        local_path='')
+        save_path='')
     assert_index_equal(expected_index, data.index)
     assert 'ghi' in data.columns
     assert 'dni_std' in data.columns
     assert 'dhi_min' in data.columns
     assert 'lwd_max' in data.columns
     assert 'relative_humidity' in data.columns
+    # test that a local file was saved and is read correctly
+    data2, metadata2 = read_bsrn('tam0616.dat.gz')
```

Comment:
Minor nitpick: this `tam0616.dat.gz` is left behind after running the tests and git thinks it's an untracked file.  Could we delete that file after the test is run, either automatically through a [temporary directory](https://docs.python.org/3/library/tempfile.html#tempfile.TemporaryDirectory) or manually?

Reply:
Done!